src/locationTracker/mapTracker/Marker.tsx

```diff
@@ -42,6 +44,7 @@ export function Marker({
     tooltip?: React.ReactNode;
     onClick: (ev: TriggerEvent) => void;
     onContextMenu?: (ev: React.MouseEvent) => void;
+    onItemDrag: (params: ItemData) => void;
```

Comment:
```suggestion
    onItemDrag?: (params: ItemData) => void;
```

then you don't need the no-ops in the non-droppable markers.